packages/library-base/src/commonMain/kotlin/io/realm/Configuration.kt

```diff
@@ -142,54 +140,20 @@ public interface Configuration {
         protected var compactOnLaunchCallback: CompactOnLaunchCallback? = null
 
         /**
-         * Creates the RealmConfiguration based on the builder properties.
-         *
-         * @return the created RealmConfiguration.
-         */
-        public abstract fun build(): T
-
-        /**
-         * Sets the path to the directory that contains the realm file. If the directory does not
-         * exists, it and all intermediate directories will be created.
-         *
-         * If not set the realm will be stored at the default app storage location for the platform:
-         * ```
-         * // For Android the default directory is obtained using
-         * Context.getFilesDir()
-         *
-         * // For JVM platforms the default directory is obtained using
-         *  System.getProperty("user.dir")
-         *
-         * // For macOS the default directory is obtained using
-         * platform.Foundation.NSFileManager.defaultManager.currentDirectoryPath
+         * Sets the filename of the realm file.
          *
-         * // For iOS the default directory is obtained using
-         * NSFileManager.defaultManager.URLForDirectory(
-         *      NSDocumentDirectory,
-         *      NSUserDomainMask,
-         *      null,
-         *      true,
-         *      null
-         * )
-         * ```
+         * If setting the full path of the realm, this name is not taken into account.
          *
-         * @param directoryPath either the canonical absolute path or a relative path from the current directory ('./').
+         * @throws IllegalAttributeException if the name includes a path separator.
```

Comment:
What is `IllegalAttributeException`? Shouldn't this be `IllegalArgumentException`?